website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -6,7 +6,7 @@ import { FlaggableElement } from "src/components/FlaggableElement";
 interface Message {
   text: string;
   id: string;
-  message_id: string;
+  frontend_message_id: string;
```

Comment:
You'll need to merge, I just merged in a change that I think fixes this.

Reply:
Thank you very much for your review.